lib/hanami/environment.rb

```diff
@@ -140,12 +140,14 @@ class Environment
     # When initialized, it sets standard `ENV` variables for Rack and Hanami,
     # such as `RACK_ENV` and `HANAMI_ENV`.
     #
-    # It also evaluates configuration from `.env` and `.env.<environment>`
+    # It evaluates configuration ONLY from `.env.<environment>` file
     # located under the config directory. All the settings in those files will
     # be exported as `ENV` variables.
     #
-    # The format of those `.env` files is compatible with `dotenv` and `foreman`
-    # gems.
+    # Master `.env` file is ignored for security reason.
```

Comment:
Will people lacking the context of the discussions that lead to this PR understand this?